druid/ingestion-spec/rollups/tpd_index_hourly_syncts.json

```diff
@@ -0,0 +1,176 @@
+              "expr": "$.object.rollup.l1"
+            },
+            {
+              "type": "path",
+              "name": "user_signin_type",
+              "expr": "$.userdata.usersignintype"
+            },
+            {
+              "type": "path",
+              "name": "actor_id",
+              "expr": "$.actor.id"
+            },
+            {
+              "type": "path",
+              "name": "context_did",
+              "expr": "$.context.did"
+            }
+          ]
+        },
+        "dimensionsSpec": {
+          "dimensions": [
+            {
+              "type": "root",
+              "name": "eid"
+            },
+            {
+              "type": "string",
+              "name": "context_env"
+            },
+            {
+              "type": "string",
+              "name": "user_signin_type"
+            },
+            {
+              "type": "string",
+              "name": "collection_type"
+            },
+            {
+              "type": "string",
+              "name": "edata_id"
+            },
+            {
+              "type": "string",
+              "name": "edata_mode"
+            },
+            {
+              "type": "string",
+              "name": "edata_pageid"
+            },
+            {
+              "type": "string",
+              "name": "edata_type"
+            },
+            {
+              "type":"string",
+              "name": "first_time_user"
+            }
+          ]
+        },
+        "timestampSpec": {
+          "column": "syncts",
+          "format": "auto"
+        }
+      }
+    },
+    "transformSpec": {
+      "transforms": [
+        {
+          "type": "expression",
+          "name": "first_time_user  ",
+          "expression": "if((\"context_cdata_type\" == 'FirstTimeUser') && \"context_cdata_id\" == 'true') , 'true','false')"
+        },
+        {
+          "type": "expression",
+          "name": "filter_pdata_id",
+          "expression": "if((like(\"context_pdata_pid\",'%contentplayer%'), if ( \"eid\" == 'START', 'true','false') ,'true')"
+        }
+      ],
+      "filter": {
+        "type": "selector",
+        "dimension": "filter_pdata_id",
+        "value": "true"
+      }
+    },
+    "metricsSpec": [
+      {
+        "name": "total_count",
+        "type": "count"
+      },
+      {
+        "type": "HLLSketchBuild",
+        "name": "unique_devices",
+        "fieldName": "dimensions_did"
+      },
+      {
+        "type": "HLLSketchBuild",
+        "name": "unique_users",
+        "fieldName": "actor_id"
+      }
+    ],
+    "granularitySpec": {
+      "type": "uniform",
+      "segmentGranularity": "hour",
+      "queryGranularity": "hour",
+      "rollup": true
+    }
+  },
+  "ioConfig": {
+    "topic": "{{env}}.druid.events.telemetry",
+    "consumerProperties": {
+      "bootstrap.servers": "{{kafka_brokers}}"
+    },
+    "taskCount": 1,
+    "replicas": 1,
+    "taskDuration": "PT14400S",
```

Comment:
Please change the task duration to 2 hours. Also, please add a publishing timeout to the ioConfig.